### PR TITLE
Modify rule S6390: Change text to the progressive education format (APPSEC-427)

### PR DESCRIPTION
**Pull request by gaetan-ferry-sonarsource**

### CODE REVIEW

File: rules/S6390/java/how-to-fix-it/java-se.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        }
+    }
+----
+
+=== How does this work?
+
+In most cases, it is discouraged to define a thread suspension time from
+user-input.
+
+If really necessary, the application should ensure that the provided suspension
```
Comment (egon-okerman-sonarsource) [inline rules/S6390/java/how-to-fix-it/java-se.adoc:38]:
```suggestion
If necessary, the application should ensure that the provided suspension
```

File: rules/S6390/java/how-to-fix-it/java-se.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+time is below a safe limit. This one should be evaluated and set to the lowest
+possible time to ensure the application's operation and limit denial of service
+attacks.
```
Comment (egon-okerman-sonarsource) [inline rules/S6390/java/how-to-fix-it/java-se.adoc:41]:
```suggestion
time is below a safe limit. Such a limit should be evaluated and set to the lowest
possible time that ensures the application's operation and restricts denial of service
attacks.
```

**Review by egon-okerman-sonarsource — APPROVED** [review]
I've added two small suggestions on how to word some sentences better, but regardless of those changes LGTM :)

File: rules/S6390/java/how-to-fix-it/java-se.adoc (hunk 1 above)
Comment (gaetan-ferry-sonarsource) [reply rules/S6390/java/how-to-fix-it/java-se.adoc:38]: I would prefer living the really here, even if grammarly does not agree. I really want to emphasize that you should **REALLY** consider not doing.

**Pull request merged**